plutus-pab/app/Cli.hs

```diff
@@ -154,30 +155,30 @@ runConfigCommand trace _ Config {..} serviceAvailability (MockWallet) =
         serviceAvailability
 
 -- Run mock node server
-runConfigCommand trace _ Config {nodeServerConfig} serviceAvailability MockNode =
+runConfigCommand trace _ Config {nodeServerConfig} serviceAvailability _ MockNode =
     liftIO $ NodeServer.main
         (toMockNodeServerLog trace)
         nodeServerConfig
         serviceAvailability
 
 -- Run mock metadata server
-runConfigCommand trace _ Config {metadataServerConfig} serviceAvailability Metadata =
+runConfigCommand trace _ Config {metadataServerConfig} serviceAvailability _ Metadata =
     liftIO $ Metadata.main
         (toMetaDataLog trace)
         metadataServerConfig
         serviceAvailability
 
 -- Run PAB webserver
-runConfigCommand trace _ config@Config{pabWebserverConfig} serviceAvailability PABWebserver =
+runConfigCommand trace _ config@Config{pabWebserverConfig} serviceAvailability eventfulBackend PABWebserver =
         fmap (either (error . show) id)
-        $ App.runApp (toPABMsg trace) config
+        $ App.runApp eventfulBackend (toPABMsg trace) config
         $ do
             App.AppEnv{App.walletClientEnv} <- Core.askUserEnv @ContractExe @App.AppEnv
             (mvar, _) <- PABServer.startServer pabWebserverConfig (Left walletClientEnv) serviceAvailability
             liftIO $ takeMVar mvar
 
 -- Fork a list of commands
-runConfigCommand trace logConfig config serviceAvailability (ForkCommands commands) =
+runConfigCommand trace logConfig config serviceAvailability backend (ForkCommands commands) =
```

Comment:
This applies for all occurrences of `backend` - could you use a more telling name instead? Probably just `eventfulBackend`, like above. Because `backend` could really be anything